### DISCUSSION TITLE
Stack toast messages

### DISCUSSION THREAD

**Discussion by kerkness** [livewire/flux#910] [Feature Requests] [answered]
Just a little feature request.  Sometimes in my app the user will get two or three notificiations in fairely quick succession about 1 or 2 seconds apart.  

It would be handy if the Toast component allowed for more than one message at a time so if a second message is necessary before the first message timesout they stack on top of each other.

I use Notistack in a React app and it works very well for this usecase. 
https://notistack.com/features/basic

The library provides a "maximum snackbars" property that lets your push  notification messages onto the "stack".  A similar feature in `flux:toast`  might be something to consider.

**Comment by jeffchown** [comment, marked as answer]
Hi @kerkness. See https://github.com/livewire/flux/discussions/137

**Reply by kerkness** [reply]
Thanks. I had searched in issues but not discussions
